docs/samples/s3/s3_secret.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: mysecret
+  annotations:
+     serving.kubeflow.org/s3-endpoint: minio-service.kubeflow:9000 # replace with your s3 endpoint
```

Comment:
Is minio-service already deployed somewhere in some kubeflow installations?

Reply:
yes that's part of the kubeflow installation.